modules/infinite-scroll.php

```diff
@@ -215,23 +215,6 @@ public function action_wp_enqueue_scripts() {
 		if ( Jetpack::is_module_active( 'tiled-gallery' ) ) {
 			Jetpack_Tiled_Gallery::default_scripts_and_styles();
 		}
-
-		// Core's Audio and Video Shortcodes
```

Comment:
Currently, IS loads `wp-mediaelement` eagerly on all pages, which is inefficient. Also, the `wp-mediaelement` script depends on `jquery`, which makes matters even worse.

WordPress will enqueue `wp-mediaelement` when a media element that requires it (e.g. `[audio]`) appears on any particular page. Infinite scroll is responsible for loading these assets only when it injects any media elements on subsequent pages. This and the related fixes ([1](https://github.com/Automattic/jetpack/pull/15017/files#diff-a56d2e4ea84f298403a898fbd92f8e12R1065-R1068), [2](https://github.com/Automattic/jetpack/pull/15017/files#diff-a56d2e4ea84f298403a898fbd92f8e12R1648-R1665)) accomplish just that.